src/efu/test/ParserTest.cpp

```diff
@@ -16,9 +16,10 @@ static int dummy_command(std::vector<std::string>, char *, unsigned int *) {
 // clang-format off
 std::vector<std::string> commands {
   "STAT_GET_COUNT",                 "STAT_GET_COUNT 1",
-  "CMD_GET_COUNT",                  "CMD_GET_COUNT 9",
+  "CMD_GET_COUNT",                  "CMD_GET_COUNT 11",
   "STAT_GET 1",                     "STAT_GET dummystat 42",
   "STAT_GET 2",                     "STAT_GET  -1",
+  "CALIB_MODE_GET",                 "CALIB_MODE_GET 0",
   "EXIT",                           "<OK>"
```

Comment:
Shouldn't the CALIB_MODE_SET added here?

Reply:
That would go in a different test. I am probably not going to implement that